jiracli/templates.go

```diff
@@ -433,18 +441,27 @@ fields:
   assignee: {{if .fields.assignee.name}}
     emailAddress: {{ or .fields.assignee.name}}
   {{- else }}
-    emailAddress: {{.fields.assignee.emailAddress}}{{end}}{{end}}{{end}}
+    emailAddress: {{.fields.assignee.emailAddress}}
+  {{end}}{{end}}
+{{end}}
```

Comment:
Keep `{{-` or `-}}` to avoid the extra blank lines.
```suggestion
  {{-end}}{{end}}
{{-end}}
```